tests/ecs/ambiguity_detection.rs

```diff
@@ -41,49 +26,24 @@ pub fn main() {
     app.cleanup();
     app.update();
 
-    let sub_app = app.main();
-
-    let ignored_schedules = get_ignored_ambiguous_system_schedules();
-
-    let ambiguities = count_ambiguities(sub_app);
-    let mut unexpected_ambiguities = vec![];
-    for (&label, &count) in ambiguities.0.iter() {
-        if ignored_schedules
-            .iter()
-            .any(|label_to_ignore| **label_to_ignore == *label)
-        {
-            continue;
-        }
-        if count != 0 {
-            unexpected_ambiguities.push(label);
-        }
-    }
+    let main_app_ambiguities = count_ambiguities(app.main());
     assert_eq!(
-        unexpected_ambiguities.len(),
-        0,
-        "Main app has unexpected ambiguities among these schedules: {:?}.\n\
-    More Details:\n{:#?}",
-        unexpected_ambiguities,
-        ambiguities
-    );
-
-    let total_ambiguities = ambiguities.total();
-    assert_eq!(
-        total_ambiguities, 72,
-        "Main app does not have an expected conflicting systems count, \
-        you might consider verifying if it's normal, or change the expected number.\n\
-        Details:\n{:#?}",
-        ambiguities
+        main_app_ambiguities.total(),
+        // This number *should* be zero.
+        // Over time, we are working to reduce the number: your PR should not increase it.
+        // See https://github.com/bevyengine/bevy/issues/7386 for progress.
+        72,
+        "Main app has unexpected ambiguities among the following schedules: \n {:?}.",
```

Comment:
```suggestion
        "Main app has unexpected ambiguities among the following schedules: \n{:#?}.",
```
Probably easier to read